### ISSUE TITLE
RawSqlBuilder columnMapping is confused by commas in COALESCE

### ISSUE THREAD

**Issue by Tyde** [ebean-orm/avaje-ebeanorm#540]
My following query starts with 
```sql
SELECT c.category_id,
  c.category_name,
  c.category_parent,
  c.user_id,
  budgets.budget_id,
  budgets.category_id,
  budgets.amount as  budget,
  COALESCE(month_sums.sum,0.0) as transaction_sum,
  COALESCE(month_balances.balance,0.0) as balance,  
  COALESCE(month_sums.end_date,date_trunc('month',budgets.month),month_balances.end_date) as data_month
```
Now I want to map this with a column mapping but the created RawSql seems to have a problem with the commas inside the COALESCE brackets:

![image](https://cloud.githubusercontent.com/assets/1103207/12538157/b5f1d394-c2d2-11e5-90a5-c8ba159e1822.png)

**Comment by rbygrave** [comment]
RawSql can be parsed or unparsed ... Try using the unparsed option.

Cheers, Rob.

**Comment by rbygrave** [comment]
So instead of:

RawSqlBuilder.parse(sql);

Use:

RawSqlBuilder.unparsed(sql);

**Comment by Tyde** [comment]
Thanks. I used a view instead now, so I won't need it anymore. But it is good to know in the future.

**Comment by rbygrave** [comment]
This was actually fixed in 6.14.1 as part of pull request #510 - Added support for custom database functions that have multiple arguments 

I'll close this (as fixed in 6.14.1).
